src/configure/model/templateModels.ts

```diff
@@ -17,10 +17,10 @@ export interface PipelineTemplate {
 
 export interface PipelineTemplateMetadata {
     templateId: string;
-    label: string;
+    templateDescription: string;
```

Comment:
Okay I see you have renamed this. But while logging telemetry, I still think you should append target kind if it exists